iati_account_web/account/views.py

```diff
@@ -56,8 +60,9 @@ def onboarding(request: HttpRequest) -> HttpResponse:
     HttpResponse
     """
 
-    if not request.user.is_authenticated:
-        return redirect("oidc_authentication_init")
+    preflight = preflight_checks(request, check_onboarding=False)
+    if preflight.not_okay_to_continue:
```

Comment:
Extremely minor point: I'm not suggest we change this now (or even soon), but maybe at some point it might make sense to have it return `okay_to_continue` which the test can then `not`, as it might read a little clearer and also match patterns found in other libraries.